client/grpc/tmservice/query.pb.gw.go

```diff
@@ -20,6 +20,7 @@ import (
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/grpclog"
+	"google.golang.org/grpc/metadata"
```

Comment:
do all/most pb.gw.go files need to be reverted? @amaurym you may know the answer here?

Reply:
Yeah i would revert this. But then there are a lot of other stuff that should be reverted too (cosmovisor tests, Makefile, flags.BroadcastSync...)

Reply:
reverted